.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  extends: 'nava',
+  env: {
+    mocha: true
+  }
+}
```

Comment:
npm will always run a pretest script if you've defined one before a test script - see package.json (`"pretest": "eslint test util scripts index.js"`)